boxsdk/object/base_object.py

```diff
@@ -207,9 +207,12 @@ def _paging_wrapper(self, url, starting_index, limit, factory=None):
         :type factory:
             `callable` or None
         :returns:
-            A generator of instances returned by the given factory callable.
+            A generator of 3-tuples. Each tuple contains:
+            1) An instance returned by the given factory callable.
+            3) The number of object returned by the last paged API call
+            2) Index the current instance in the current page
```

Comment:
This is unclear. Did you index them incorrectly, or did you put the lines in the wrong order?